tox.ini

```diff
@@ -1,25 +1,27 @@
 [tox]
-envlist = 
-    py27,py36,py37,py38,py39,pypy,cover
+envlist =
+    py27,py36,py37,py38,py39,pypy27,pypy3,coverage
 
 [testenv]
-commands = 
-    python setup.py dev
-    python setup.py test
-deps =
-    py27: pyramid<=2.0
-    py27: mock<=4.0
-    py27: waitress<=2.0
+extras =
+    tests
+setenv =
+    COVERAGE_FILE=.coverage.{envname}
+commands =
+    python --version
+    coverage run -m pytest {posargs:}
```

Comment:
Used `coverage run` rather than `pytest --cov` as:

 * `pytest --cov` aborts if the coverage level is too low, rather than just outputting a file to be combined later
 * One less dependency